js/stringProcessing/getSentences.js

```diff
@@ -83,12 +87,75 @@ var splitOnIndex = function( positions, text ) {
  */
 var findSubheadings = function( text ) {
 	var subheadings = getSubheadings( text );
-	return map ( subheadings, function( subheading ) {
+	return map( subheadings, function( subheading ) {
 		return subheading.index + subheading[ 0 ].length;
 	} );
 };
 
 /**
+ * Matches a partial tag. If a sentence starts with a tag it should end with it. If it doesn't
+ * it is a partial tag and it should be removed since it can break markers.
+ * @param {string} sentence The sentence to check for tags.
+ * @returns {{startTag: string, endTag: string}} The start and endtag. If no tags are found, both return an empty string.
+ */
+var matchPartialTag = function( sentence ) {
+
+	// Matches a starttag at the beginning of the sentence.
+	var beginMatch = sentence.match( /^<(.\S|>+)/ ) || [];
+
+	// Matches an endtag at the end of the sentence.
+	var endMatch = sentence.match( /\/(.\S+)>$/ ) || [];
+
+	var startTag = "";
+	var endTag = "";
+	if ( !isUndefined( beginMatch.length > 1 ) ) {
+		startTag = beginMatch[ 1 ];
+	}
+	if ( !isUndefined( endMatch.length > 1 ) ) {
+		endTag = endMatch[ 1 ];
+	}
+	return{
```

Comment:
Missing a space between `return` and `{` and please add a newline above `return`